### PR TITLE
Adding prometheus alerts

### PR DESCRIPTION
**Pull request by rfuelsh**
- add pod and node prometheus alerts 
- integrated and sent alerts to slack
- updates to the tfe code
- updates to create-k8s.sh script

### CODE REVIEW

File: scripts/create-k8s.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ sleep 600
 echo "Now setting up your k8s cluster ..."
 
 if [ "${k8s_provider}" == "eks" ]; then
-    cd ../../../ingress/
+    cd ../ingress/
```
Comment (Voxelot) [inline scripts/create-k8s.sh:30]: looks like this will fix a bug that would prevent a future cluster deployment, nice catch

**Review by Voxelot — DISMISSED** [review]

**Comment by Voxelot** [comment]
Do we have a testing environment setup for these changes?

File: scripts/create-k8s.sh (hunk 1 above)
Comment (rfuelsh) [reply scripts/create-k8s.sh:34]: yes I am testing the create-k8s.sh against fuel-deployment and make modifications in this PR

**Comment by rfuelsh** [comment]
> Do we have a testing environment setup for these changes?

1) fuel-dev1 has the prometheus changes 

2) fuel-prod-1 is being deployed via updates to create-k8s.sh -> in progress

**Review by Voxelot — APPROVED** [review]

**Review by Voxelot — APPROVED** [review]

**Pull request merged**